articles/server-platforms/rails/00-introduction.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Introduction
+description: This tutorial will show you how to integrate Auth0 in your Rails app to add authentication and authorization.
+---
+
+## Introduction
+This is the very beginning of a simple, practical and multi-step quickstart that will guide you through managing authentication in your Ruby on Rails apps with Auth0.
+
+::: panel-info System Requirements
+This tutorial and seed project have been tested with the following:
+* Ruby 2.3.1
+* Rails 5.0.0
+:::
+
+<%= include('../../_includes/_package', {
+  pkgRepo: 'omniauth-auth0',
+  pkgBranch: 'master',
+  pkgPath: 'examples/ruby-on-rails-webapp',
+  pkgFilePath: null,
+  pkgType: 'server'
+}) %>
+
+### Seed & Samples
+There are two options to following along this quickstart. You can either download the [seed project](https://github.com/auth0-samples/auth0-rubyonrails-sample/tree/master/00-Starter-Seed) or the [samples](https://github.com/auth0-samples/auth0-rubyonrails-sample) provided at each page of this quickstart.
+
+The seed is a regular RoR app, with all the Auth0 dependencies set, but nothing more. It's an empty canvas meant to be filled as you follow along the steps of this quickstart. If you prefer this option download the seed from our [GitHub repository](https://github.com/auth0-samples/auth0-rubyonrails-sample/tree/master/00-Starter-Seed) and follow along.
+
+Instead you can choose to follow the samples that are included in each step. Each sample uses the seed project as a starting point and applies to it the configuration of each step, so for example the Login sample would be the seed project plus the configuration required to implement login functionality. If you choose to follow this approach continue reading, the rest of this document will guide you through setting up the required prerequisites.
+
+::: panel-info System Requirements
+This tutorial and seed project have been tested with the following:
+* Ruby 2.3.1
+* Rails 4.2.6
+:::
+
+<%= include('../../_includes/_package', {
+  pkgRepo: 'omniauth-auth0',
+  pkgBranch: 'master',
+  pkgPath: 'examples/ruby-on-rails-webapp',
+  pkgFilePath: null,
+  pkgType: 'server'
+}) %>
```

Comment:
Seems like this is duplicated? This should also point to the starter seed in auth0-samples
https://github.com/auth0-samples/auth0-rubyonrails-sample/tree/master/00-Starter-Seed